### ISSUE TITLE
Can function be added to detect if Compton compositor is running and shut it down first if true?

### ISSUE THREAD

**Issue by stevenpusser** [compiz-reloaded/fusion-icon#2]
Title pretty much says it all.  Current behavior is just that the window decorations vanish, leaving newbies in a state of bewilderment.

**Comment by soreau** [comment]
I don't think we should special-case this, at least not in a hard-coded fashion. More interesting to know might be 'why' this happens.
